typescript/sdk/src/consts/multisigIsm.ts

```diff
@@ -189,4 +189,22 @@ export const defaultMultisigIsmConfigs: ChainMap<MultisigIsmConfig> = {
       '0xdf181fcc11dfac5d01467e4547101a856dd5aa04',
     ],
   },
+  solanadevnet: {
+    type: ModuleType.LEGACY_MULTISIG,
```

Comment:
I thought we couldn't use legacy because we can't fit merkle proofs in tx data? what am I missing

Reply:
Iiuc this is the configuration of the solanadevnet set that will be created on all other chains, so it indicates what type of ISM on all chains other than solanadevnet should be created to receive messages from solanadevnet. Messages from solanadevnet can still be processed with a merkle proof

It's legacy_multisig because all others are - didn't wanna choose something else bc I didn't wanna unnecessarily risk introducing more potentially untested features

Reply:
ended up just removing this, realized it's not needed if I'm just running the infra